electron-app/src/components/accounts/AccountBalances.vue

```diff
@@ -107,20 +107,18 @@
       <template v-if="balances.length > 0" #body.append>
         <tr class="account-balances__total">
           <td>Total</td>
-          <td>
-            {{
-              visibleBalances.map(val => val.amount)
-                | balanceSum
-                | formatPrice(floatingPrecision)
-            }}
+          <td class="text-end">
+            <amount-display
+              :value="visibleBalances.map(val => val.amount) | balanceSum"
+            >
+            </amount-display>
           </td>
-          <td>
-            {{
-              visibleBalances.map(val => val.usdValue)
-                | balanceSum
-                | calculatePrice(exchangeRate(currency.ticker_symbol))
-                | formatPrice(floatingPrecision)
-            }}
+          <td class="text-end">
+            <amount-display
+              usd-value
```

Comment:
Don't forget that I renamed that to fiat now so you will have to replace `usd-value` -> `fiat`

Reply:
Yeah I missed it in a few places, fixing on my end.